retrieve_tos_content.py

```diff
@@ -0,0 +1,30 @@
+import time
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.chrome.options import Options
+from bs4 import BeautifulSoup
+
+def fetch_tos_content(url):
```

Comment:
This function is not intended to be called from outside of this file right? In that case you should make it `private`. Unfortunately Python does not have a way to enforce scope of functions, but there is a widely adopted consensus to prefix private functions or methods with `_` so everybody who read the code knows that this function should not be called from outside.

Please rename it to `_fetch_tos_content`.